app-config.yaml

```diff
@@ -30,6 +30,14 @@ proxy:
       Circle-Token:
         $env: CIRCLECI_AUTH_TOKEN
 
+  '/splunk-on-call':
```

Comment:
Again, would suggest we may not want to change this globally for the example app, but if needed adding it to the README for the plugin setup instructions?